core/src/main/java/io/keploy/regression/keploy/Filter.java

```diff
@@ -7,11 +7,13 @@
 @Setter
 @NoArgsConstructor
 public class Filter {
-    private String UrlRegex;
+    private String acceptUrlRegex;
```

Comment:
`acceptUrlRegex` should also be String[],

Reply:
so Filter class should contain 
String[] acceptUrlRegex;
Sting[] acceptHeaderRegex;
Sting[] rejectHeaderRegex;
Sting[] rejectUrlRegex;

Reply:
Yes